### PR TITLE
[keymap]kbdfans/kbd8x_mk2 add ISO 88keys

### PR DESCRIPTION
**Pull request by TecHunter**
## Description

Added kbdfans/kbd8x_mk2 keymap for ISO layout with 88keys.
Tested with swiss french keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kbdfans/kbd8x_mk2/keymaps/iso_88keys/readme.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# The default keymap for KBD8X MKII
```
Comment (noroadsleft) [inline keyboards/kbdfans/kbd8x_mk2/keymaps/iso_88keys/readme.md]:
```suggestion
# iso_88keys keymap for KBD8X MKII
```

**Comment by noroadsleft** [comment]
Much better, here. :+1:

**Comment by fauxpark** [comment]
This might be more useful if it were added as a proper layout macro, instead of just a keymap. It would require renaming `LAYOUT` to `LAYOUT_tkl_ansi` and aliasing it to `LAYOUT`, creating `LAYOUT_tkl_iso`, and fixing the default keymap JSON in the Configurator.

**Comment by TecHunter** [comment]
> This might be more useful if it were added as a proper layout macro, instead of just a keymap. It would require renaming `LAYOUT` to `LAYOUT_tkl_ansi` and aliasing it to `LAYOUT`, creating `LAYOUT_tkl_iso`, and fixing the default keymap JSON in the Configurator.

I can do this later, I'm new in this..

**Pull request closed without merging**